src/library_wasmfs.js

```diff
@@ -285,6 +285,12 @@ FS.createPreloadedFile = FS_createPreloadedFile;
     // TODO: truncate
     // TODO: ftruncate
     // TODO: utime
+    utime: (path, atime, mtime) => {
+      return FS.handleError(withStackSave(() => {
+        var pathBuffer = stringToUTF8OnStack(path);
+        return __wasmfs_utime(pathBuffer, atime, mtime, 0);
```

Comment:
As this is a function for our internal use, I don't think we need to add the last parameter which is always 0 in our usage. (If we need it later, we can add it later - it's ok to change internal APIs.)